gaplint.py

```diff
@@ -1720,15 +1751,20 @@ def __get_yml_dict() -> Tuple[str, Dict[str, Any]]:
     if config_yml_fname is None:
         return "", {}
 
-    _info_action(f"Using configurations in {config_yml_fname}")
     try:
         with open(
             config_yml_fname, "r", encoding="utf-8", newline=None
         ) as config_yml_file:
             yml_dic = yaml.load(config_yml_file, Loader=yaml.FullLoader)
-    except (yaml.YAMLError, IOError):
-        _info_action("IGNORING {config_yml_fname}: error parsing YAML")
+    except yaml.YAMLError:
+        _info_action(
+            f"IGNORING config file {config_yml_fname}: error parsing YAML"
+        )
         return "", {}
+    except IOError as e:
+        _info_action(f"IGNORING config file {config_yml_fname}:\n  {e}")
```

Comment:
```suggestion
        _info_action(f'IGNORING config file "{config_yml_fname}" because of the following IO error:\n  {e}')
```
Maybe changing the error message here would make it clearer to the user what the issue is when an `IOError` occurs (such as e.g. when passing the empty string as a path). Otherwise it appears as if some sort of uncaught error has occured.